data/minio/bucket/custom_files/training_params.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-WORLD_NAME: "LGSWide"
+WORLD_NAME: "Bowtie_track"
```

Comment:
Please change this to default to the Reinvent 2018 track

Reply:
I have accepted incoming master change. Track is now LGSWide and OBJECT_AVOIDANCE mode.